.github/SCOPES.md

```diff
@@ -0,0 +1,24 @@
+## Scopes
```

Comment:
What happens if something is not on this list? For instance, a change to the CI, to the github templates, the the commit for this particular issue?